packages/client/utils/parseEmailAddressList.ts

```diff
@@ -11,7 +12,20 @@ const parseEmailAddressList = (rawStr = '') => {
     .trim()
     // remove trailing commas
     .replace(/,$/g, '')
-  return emailAddresses.parseAddressList(commaDelimStr)
+
+  const commaDelimArr = commaDelimStr.split(', ')
```

Comment:
True! I've removed the spaces and pushed the update. 

For the alternate solution, if we had a raw string of "validEmail1, invalidEmail, validEmail2" and it returned null, if we reduce the raw string by 1 character until it's no longer null, wouldn't we be left with only "validEmail1", i.e. we'd lose "validEmail2"?

Reply:
that's correct, i think it's OK if we say something like "invalid email(s) after validEmail1" because if they goof on the syntax then it's possible that goof is what made validEmail2 valid. e.g. `matt@parabol.co,b@ b@b.co`. we can't guarantee that they meant `b@b.co`, maybe they meant bb@b.co?

Reply:
Fair point! I've added the `invalid email(s) after validEmail1` feedback while preventing the jumping. It's ready for a re-review when you are